client/src/main/js/factory/Local.js

```diff
@@ -25,18 +25,54 @@
             Object.assign(this, local);
         }
 
+        /**
+         * Adiciona um equipamento ao local.
+         *
+         * @param equipamento Equipamento a ser adicionado.
+         */
         Local.prototype.addEquipamento = function (equipamento) {
             this.equipamentos.push(equipamento);
         };
 
-        Local.prototype.getFuncionamentoFormatado = function () {
+        /**
+         * Retorna o horário de início do funcionamento, no formato hh:mm, do local.
+         *
+         * @return {string} Horário de início do funcionamento do local.
+         */
+        Local.prototype.getInicioFuncionamento = function () {
             if (this.funcionamento) {
                 const horaInicio = this.funcionamento.substring(PRIMEIRO_INDICE, HORA_INICIO_INDICE);
                 const minutoInicio = this.funcionamento.substring(HORA_INICIO_INDICE, MINUTO_INICIO_INDICE);
+
+                return `${horaInicio}:${minutoInicio}`;
```

Comment:
existe um método do moment que dá pra pegar direto o HH:mm. Olha em dataManipuladorService#getHorarioEmString

Reply:
O this.funcionamento é uma string do tipo: 08001800 (Corresponde a um início às 8h e término às 18h)